examples/visual-tests/amp-autocomplete/amp-autocomplete.amp.html

```diff
@@ -54,7 +59,7 @@ <h2>Rich content, remote data</h2>
       filter="prefix" filter-value="countryName" min-characters="0">
       <input type="search" id="input2">
       <template type="amp-mustache">
-        <div value="{{countryName}}">
+        <div data-value="{{countryName}}">
```

Comment:
Is this an API change or was this broken before?

Reply:
Broken before 😎

Reply:
sweet 🤙